build/grunt.js

```diff
@@ -114,14 +114,6 @@ module.exports = function(grunt) {
       build: ['build/temp/*'],
       dist: ['dist/*']
     },
-    jshint: {
```

Comment:
This is a temporary step. The final PR in this series will restore automatic linting capabilities, but for now we want them off.